lib/qbo_api/configuration.rb

```diff
@@ -40,5 +40,14 @@ def minor_version
     def minor_version=(value)
       @minor_version = value
     end
+
+    def refresh_proc
+      @refresh_proc
+    end
+
+    def refresh_proc=(value)
+      @refresh_proc = value
```

Comment:
should be in oauth2 imho since it's oauth2 specific

the point of the file is to actually fully separate them